src/domain/collection/CollectionPageContainer.tsx

```diff
@@ -82,6 +85,24 @@ const CollectionPageContainer: React.FC = () => {
     return null;
   };
 
+  const scrollToEventCard = (id: string) => {
+    document
+      .getElementById(id)
+      ?.scrollIntoView({ block: 'center', behavior: 'smooth' });
+  };
+
+  // Scroll back to correct event card (event tat user is coming back from)
+  React.useEffect(() => {
+    if (location.state?.eventId) {
+      const cardId = getLargeEventCardId(location.state.eventId);
+      scrollToEventCard(cardId);
+      // Clear eventId value to keep scroll position correctly
+      const state = { ...location.state };
+      delete state.eventId;
+      history.replace({ ...location, state });
```

Comment:
Not sure in which situation this is really needed, it is same as in EventSearchPageContainer.tsx

Reply:
Found this with a quick search: https://stackoverflow.com/a/47513369/784642

Reply:
I was just thinking in what situation this could cause problems if left out...

Reply:
Without this if I would navigate back and forward I would still get restored scroll, not sure if that would be problematic